pkg/virt-controller/services/template_test.go

```diff
@@ -4361,27 +4361,34 @@ var _ = Describe("Template", func() {
 				migrationName = "testmigration"
 			)
 
-			expectStateMounts := func(pod *k8sv1.Pod) {
-				Expect(pod.Spec.Volumes).To(ContainElement(k8sv1.Volume{
+			expectStateMounts := func(pod *k8sv1.Pod, enabled bool) {
+				volume := k8sv1.Volume{
 					Name: "vm-state",
 					VolumeSource: k8sv1.VolumeSource{
 						PersistentVolumeClaim: &k8sv1.PersistentVolumeClaimVolumeSource{
 							ClaimName: claimName,
 						},
 					},
-				}))
-				Expect(pod.Spec.Containers[0].VolumeMounts).To(ContainElements(
-					k8sv1.VolumeMount{
-						MountPath: "/var/lib/libvirt/swtpm",
-						Name:      "vm-state",
-						SubPath:   "swtpm",
-					},
-					k8sv1.VolumeMount{
-						MountPath: "/var/lib/swtpm-localca",
-						Name:      "vm-state",
-						SubPath:   "swtpm-localca",
-					},
-				))
+				}
+				mount1 := k8sv1.VolumeMount{
+					MountPath: "/var/lib/libvirt/swtpm",
+					Name:      "vm-state",
+					SubPath:   "swtpm",
+				}
+				mount2 := k8sv1.VolumeMount{
+					MountPath: "/var/lib/swtpm-localca",
+					Name:      "vm-state",
+					SubPath:   "swtpm-localca",
+				}
+				if enabled {
+					Expect(pod.Spec.Volumes).To(ContainElement(volume))
+					Expect(pod.Spec.Containers[0].VolumeMounts).To(ContainElement(mount1))
```

Comment:
expect(pod.Spec.Containers[0].VolumeMounts).To(ContainElements(mount1, mount2))?

Reply:
In this case yes, but I kept the split to highlight the fact that we're checking the same thing as the `else`, just in reverse.